src/Chapter1.hs

```diff
@@ -632,7 +644,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n | n < 0 = sumLast2 (-n)
+sumLast2 n = last1 + last2
+  where
+    last1 = n `mod` 10
+    last2 = (n `div` 10) `mod` 10
```

Comment:
That is a nice solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

I see that you have both:

```haskell
n `mod` 10
n `div` 10
```
One hint to make it shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. 

So you could write it this way:

```haskell
(x, y) = abs n `divMod` 10
```

You can see how we could pattern match on the pair 🙂